### ISSUE TITLE
Import watched states and playback progress from trakt.tv not working on Jellyfin Unstable 10.10

### ISSUE THREAD

**Issue by foux** [jellyfin/jellyfin-plugin-trakt#237]
Hello there!

I very well know that Jellyfin 10.10 is unstable, and not all plugins are supposed to work, but I report this issue so that it can be fixed before 10.10 release.

On 10.10, when running the scheduled task to import progress, I run into the following issue : 
```
[17:01:27] [ERR] [31] Trakt.ScheduledTasks.SyncFromTraktTask: Error syncing trakt.tv data for user myUser
System.MissingMethodException: Method not found: 'MediaBrowser.Controller.Entities.UserItemData MediaBrowser.Controller.Library.IUserDataManager.GetUserData(System.Guid, MediaBrowser.Controller.Entities.BaseItem)'.
   at Trakt.ScheduledTasks.SyncFromTraktTask.SyncTraktDataForUser(User user, Double currentProgress, IProgress`1 progress, Double percentPerUser, CancellationToken cancellationToken)
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine)
   at Trakt.ScheduledTasks.SyncFromTraktTask.ExecuteAsync(IProgress`1 progress, CancellationToken cancellationToken)
[17:01:27] [ERR] [31] Trakt.ScheduledTasks.SyncFromTraktTask: Error syncing trakt.tv data for user myUser2
System.MissingMethodException: Method not found: 'MediaBrowser.Controller.Entities.UserItemData MediaBrowser.Controller.Library.IUserDataManager.GetUserData(System.Guid, MediaBrowser.Controller.Entities.BaseItem)'.
   at Trakt.ScheduledTasks.SyncFromTraktTask.SyncTraktDataForUser(User user, Double currentProgress, IProgress`1 progress, Double percentPerUser, CancellationToken cancellationToken)
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine)
   at Trakt.ScheduledTasks.SyncFromTraktTask.ExecuteAsync(IProgress`1 progress, CancellationToken cancellationToken)
[17:01:27] [INF] [31] Emby.Server.Implementations.ScheduledTasks.TaskManager: Import watched states and playback progress from trakt.tv Completed after 0 minute(s) and 0 seconds
```

Sending progress from jellyfin (during normal operations : watching, marking as read, ...) works.
Thanks you!

**Comment by crobibero** [comment]
This should be fixed in the latest unstable build of the plugin `25.2410.9.0`

**Comment by foux** [comment]
Thanks! Is there a repo to install unstable from, or should it be built locally?

**Comment by crobibero** [comment]
https://repo.jellyfin.org/files/plugin-unstable/manifest.json

**Comment by foux** [comment]
Confirmed fixed on `25.2410.9.0`, thanks for the lighting fast fix @crobibero
